system-tests/src/main/java/io/apicurio/registry/systemtests/registryinfra/ResourceManager.java

```diff
@@ -201,32 +248,41 @@ public final <T extends HasMetadata> void deleteResource(T resource) {
         }
 
         Assertions.assertTrue(
-                waitResourceCondition(resource, type::doesNotExist, TimeoutBudget.ofDuration(Duration.ofMinutes(3))),
+                waitResourceCondition(resource, type::doesNotExist, TimeoutBudget.ofDuration(Duration.ofMinutes(10))),
```

Comment:
This is extremely scary, waiting 10 minutes for a resource to be deleted is going to increment the (already long) feedback cycle and possibly mask other issues.